README.md

```diff
@@ -1,99 +1,131 @@
 AWS Device Farm Gradle Plugin
 ------------------------------
 
-AWS Device Farm integration with Android Gradle Build system
+AWS Device Farm integration with the Android Gradle Build system
 
-This plugin provides [AWS Device Farm] (http://aws.amazon.com/device-farm) functionality from your Android gradle environment, allowing you to kick off tests on real Android phones and tablets hosted in the AWS Cloud.
-
-For more information see [AWS Device Farm Developer Guide] (http://docs.aws.amazon.com/devicefarm/latest/developerguide/welcome.html)
+This plugin provides [AWS Device Farm](http://aws.amazon.com/device-farm) functionality from your Android gradle environment, allowing you to kick off tests on real Android phones and tablets hosted in the AWS Cloud. 
 
+For more information see the [AWS Device Farm Developer Guide](http://docs.aws.amazon.com/devicefarm/latest/developerguide/welcome.html). 
 
 Usage
 =====
 
-## Building the plugin:
+1. Add the plugin artifact to your dependency list in `build.gradle`.
```

Comment:
It would help to mention which level gradle file is to be updated. Module or top most app level ?

Reply:
I've reworded to make it clearer that you modify the top-level `build.gradle`. 
I've also said that you can copy-paste the code snippet into the top-level `build.gradle` file if you're so inclined.